HomeWork/src/main/java/exception/ArrayValueCalculator.java

```diff
@@ -0,0 +1,50 @@
+package exception;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+public class ArrayValueCalculator {
+    String[][] matrix = new String[4][4];
+    private static final int MAX_MATRIX_SIZE = 4;
+
+    public void arrayInitialization(String[][] values) {
```

Comment:
Make this method static and replace `void` with `String[][]` - return an initialized array.